infracost-usage-example.yml

```diff
@@ -103,6 +103,9 @@ resource_usage:
   aws_nat_gateway.my_nat_gateway:
     monthly_data_processed_gb: 10 # Monthly data processed by the NAT Gateway in GB.
 
+  aws_route53_record.my_record:
+    monthly_queries: 1000000000 # Monthly number of queries (Standard, Latency Based, Geo DNS or Geoproximity)
```

Comment:
Since the pricing is [different](https://aws.amazon.com/route53/pricing/) for the three types of queries, I think we should have 3 params here and use the relevant one in the resource go file:
monthly_standard_queries
monthly_latency_based_queries
monthly_geo_queries